cmdb/framework/models/type.py

```diff
@@ -453,6 +458,8 @@ def __init__(self, public_id: int, name: str, author_id: int, render_meta: TypeR
         self.active: bool = active
         self.author_id: int = author_id
         self.creation_time: datetime = creation_time or datetime.now()
+        self.editor_id: int = editor_id
+        self.last_edit_time: datetime = last_edit_time
```

Comment:
Missing default value

Reply:
Default value is specified in the body of the constructor

Reply:
Um, no, that's the null value. Oh or is the value `last_edit_time` only set when editing once? Then it is ok like this.

Reply:
Exactly